examples/ota-requestor-app/ota-requestor-common/BDXDownloader.cpp

```diff
@@ -69,16 +75,29 @@ void BdxDownloader::HandleTransferSessionOutput(TransferSession::OutputEvent & e
         VerifyOrReturn(err == CHIP_NO_ERROR, ChipLogError(BDX, "SendMessage failed: %" CHIP_ERROR_FORMAT, err.Format()));
         break;
     }
-    case TransferSession::OutputEventType::kAcceptReceived:
+    case TransferSession::OutputEventType::kAcceptReceived: {
+        OTAImageProcessorParams params = {
+            .imageFile = outFilePath,
+            .downloadedBytes = 0,
+            // TODO: Set this to Length field of ReceiveAccept
+            .totalFileBytes = 0,
+        };
+        mImageProcessor->SetOTAImageProcessorParams(params);
+
+        err = mImageProcessor->PrepareDownload();
```

Comment:
This is a syncrhonous call. It must be asynchronous since "preparing" may take a very long time on some platforms.

The OTARequestor is what should send blocks received to the image processor.